tests/functionality_tests/obstacle_tracking_test.py

```diff
@@ -0,0 +1,195 @@
+import unittest
+import math
+from engine.robot import Robot
+from engine.mission import Mission
+from engine.base_station import BaseStation
+from engine.mission import ControlMode
+from constants.definitions import ROOT_DIR
+import random
+
+
+class TestObstacleTracking(unittest.TestCase):
+
+    def populate_standard_obstacle_data(self):
+        mock_data_path = ROOT_DIR + '/tests/functionality_tests/csv/ultrasonic_values.csv'
+        mock_data_file = open(mock_data_path, "w")
+        mock_data_file.truncate()
+        mock_data_file.write(
+            "(600, 600, 600, 600, 600, 'assuming a straight line between robot and goal with rectangular box centered in the middle')\n")
```

Comment:
maybe add a comment above w/ what each csv parameter represents for readability? like (front_value,.....)